volume/manager.go

```diff
@@ -0,0 +1,147 @@
+package volume
```

Comment:
New volume manager (service) is responsible for periodically running the task to calculate the capacity/used for volumes.